[Inductor] Add attention pattern for model DistilBert in transformers==4.44.2.

etaf opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #154110
* __->__ #154091

This PR add a attention fusion pattern that match the attention of
DistilDistilBert in transformers==4.44.2 at
https://github.com/huggingface/transformers/blob/953196a43dae6a3c474165fba7d215fcbc7b7730/src/transformers/models/distilbert/modeling_distilbert.py#L212

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov

etaf committed:
[Inductor] Add attention pattern for model DistilBert in transformers==4.44.2.

This PR add a attention fusion pattern that match the attention of
DistilDistilBert in transformers==4.44.2 at
https://github.com/huggingface/transformers/blob/953196a43dae6a3c474165fba7d215fcbc7b7730/src/transformers/models/distilbert/modeling_distilbert.py#L212

Signed-off-by: xinan.lin <xinan.lin@intel.com>

[ghstack-poisoned]

etaf commented:
@pytorchbot merge

seemethere commented:
@pytorchbot revert -c ghfirst -m "I root caused this PR to some failures, I tried to resolve with https://github.com/pytorch/pytorch/pull/154923 but it looks like there are more failures with my fix"

For a simple fix / follow-up to this all you have to do is re-run `python3 torchgen/fuse/gen_patterns.py`

Sorry for the churn here this just made things difficult to land!

etaf committed:
Update on "[Inductor] Add attention pattern for model DistilBert in transformers==4.44.2."


This PR add a attention fusion pattern that match the attention of
DistilDistilBert in transformers==4.44.2 at
https://github.com/huggingface/transformers/blob/953196a43dae6a3c474165fba7d215fcbc7b7730/src/transformers/models/distilbert/modeling_distilbert.py#L212

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[Inductor] Add attention pattern for model DistilBert in transformers==4.44.2."


This PR add a attention fusion pattern that match the attention of
DistilDistilBert in transformers==4.44.2 at
https://github.com/huggingface/transformers/blob/953196a43dae6a3c474165fba7d215fcbc7b7730/src/transformers/models/distilbert/modeling_distilbert.py#L212

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[Inductor] Add attention pattern for model DistilBert in transformers==4.44.2."


This PR add a attention fusion pattern that match the attention of
DistilDistilBert in transformers==4.44.2 at
https://github.com/huggingface/transformers/blob/953196a43dae6a3c474165fba7d215fcbc7b7730/src/transformers/models/distilbert/modeling_distilbert.py#L212

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]
